IBX-6804 - VAT rate assignment in product type (#232)
ibexa/documentation-user: docs/pim/products.md

- Virtual - non-tangible items. They can be sold individually, or as part of a product bundle.
They don't require shipment in the online process. Examples: memberships, services, warranties. 

> A product type also defines the attributes that all products of this type can have, as well as VAT rates that apply to all products of this type.

Reviewer: ```suggestion
A product type also defines both the attributes and VAT rates for all its products.
```
